aeolus/messages.h

```diff
@@ -95,7 +95,8 @@ class M_midi_info : public ITC_mesg
 {
 public:
 
-    M_midi_info (void) : ITC_mesg(MT_MIDI_INFO) {}
+    M_midi_info (void)
+        : ITC_mesg(MT_MIDI_INFO) {}
```

Comment:
Did you intend to add a newline even in cases like this, where we have a one-liner constructor with an empty body in a header file?

(I checked, and Qt keeps this all on one line.)

Reply:
I think this is not important. I will try to find options in the config for uncrustify, so that this does not happen. If I do not find such options (or someone does not tell), we will be forced to add a new line in such cases.

Reply:
I did not find an opportunity to separate behaviour for different cases.
And we discussed and decided that it is better to always wrap on a new line. One of the pluses is that often such lines do not fit into the page width of the github review.